data/alpha/field/internal-drainage-board.yaml

```diff
@@ -0,0 +1,6 @@
+cardinality: '1'
+datatype: string
+field: internal-drainage-board
+phase: alpha
+register: ''
+text: Internal drainage board code
```

Comment:
Can you add a full stop to this? All of your other field descriptions seem to, so I would assume it's good to be consistent?